src/managers/input/InputManager.cpp

```diff
@@ -784,10 +784,9 @@ void CInputManager::applyConfigToKeyboard(SKeyboard* pKeyboard) {
     if (!FILEPATH.empty()) {
         auto path = absolutePath(FILEPATH, g_pConfigManager->configCurrentPath);
 
-        if (!std::filesystem::exists(path))
+        if (FILE* const KEYMAPFILE = fopen(path.c_str(), "r"); !KEYMAPFILE)
             Debug::log(ERR, "input:kb_file= file doesnt exist");
```

Comment:
It may be more accurate now to say `cannot open file for reading`, because this check is true when the file cannot be opened for reading. It might exist but we might lack the permissions to read it, etc.